synapse/lib/stormlib/aha.py

```diff
@@ -85,6 +107,73 @@ async def _methAhaGet(self, svcname, filters=None):
         proxy = await self.runt.snap.core.reqAhaProxy()
         return await proxy.getAhaSvc(svcname, filters=filters)
 
+    async def _methCallPeerApi(self, svcname, apiname, timeout=None, skiprun=None):
```

Comment:
Rog. It just feels difficult to use from a user perspective because of the todos. And this is the first storm API where we're expecting the end user to pass in a todo. It seems like the storm API doesn't necessarily have to exactly mirror the python (or telepath) API. I'll shut up now :)

Reply:
I actually had a similar concern and wanted to make this meth a $__ private one to the Storm command. Not sure we've done that before here. I suspect most users probably won't (and maybe shouldn't?) be calling this from their runtime.

Reply:
I've refactored this to use Storm todo objects